scripts/variantstore/scripts/run_avro_query_for_sample_info.py

```diff
@@ -32,13 +32,14 @@ def get_number_of_partitions(dataset_name, project_id):
     return math.ceil(max_table_num[0])
 
 
-def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix):
+def construct_sample_info_avro_queries(call_set_identifier, dataset_name, project_id, avro_prefix, new_sample_cutoff=None):
     num_of_tables = get_number_of_partitions(dataset_name, project_id)
 
     for i in range(1, num_of_tables + 1):
         file_name = f"*.{i:03}.avro"
         id_where_clause = f"sample_id >= {((i -1) * 4000) + 1} AND sample_id <= {i * 4000}"
-
+        if new_sample_cutoff:
+            id_where_clause = f"sample_id >= {((i -1) * 4000) + 1} AND sample_id <= {i * 4000} AND sample_id < {new_sample_cutoff}"
```

Comment:
repeating question from earlier--is the comparison the right way with the final `sample_id < {new_sample_cutoff}"`
part of this clause? I might be reading this code the wrong way, but I thought the point was to only include the samples above that id in the output